lib/util/constants.js

```diff
@@ -3,23 +3,35 @@
 
 'use strict';
 
-exports.Constants = {
+var Constants = {
   constraints: ['minLength', 'maxLength', 'minimum', 'maximum', 'enum', 'maxItems', 'minItems', 'uniqueItems', 'multipleOf', 'pattern'],
   xmsExamples: 'x-ms-examples',
   exampleInSpec: 'example-in-spec',
   BodyParameterValid: 'BODY_PARAMAETER_VALID',
+  xmsSkipUrlEncoding: 'x-ms-skip-url-encoding',
+  Errors: 'Errors',
+  Warnings: 'Warnings',
   ErrorCodes: {
     InternalError: 'INTERNAL_ERROR',
+    InitializationError: 'INITIALIZATION_ERROR',
```

Comment:
As Gaurav pointed out, we should decide and finalize on how to structure these errors in terms of code and inheritance classes.